Specs/Scene/Cesium3DTileSpec.js

```diff
@@ -0,0 +1,149 @@
+/*global defineSuite*/
+defineSuite([
+        'Scene/Cesium3DTile',
+        'Core/Cartesian3',
+        'Core/defined',
+        'Core/OrientedBoundingBox',
+        'Core/Rectangle',
+        'Core/SphereOutlineGeometry',
+        'Specs/createContext',
+        'Specs/createFrameState'
+    ], function(
+        Cesium3DTile,
+        Cartesian3,
+        defined,
+        OrientedBoundingBox,
+        Rectangle,
+        SphereOutlineGeometry,
+        createContext,
+        createFrameState) {
+    "use strict";
+
+    describe('bounding volumes', function() {
+        var tileWithBoundingSphere = {
```

Comment:
I would just put all of sample headers like this at the top, outside of `describe` functions and share them among all the tests.